ts-obs/cmd/chunkInterval.go

```diff
@@ -0,0 +1,15 @@
+package cmd
```

Comment:
go file convention favors lowercase names with underscores `chunk_interval`

Reply:
Cobra specifically asks for command names in camel case, which is why I left it that way.